universal-editor-generation-variation/app.config.yaml

```diff
@@ -0,0 +1,3 @@
+extensions:
+  aem/universal-editor/1:
```

Comment:
@fe-lix- , It looks like the `aem/universal-editor/1` extension point was renamed to `universal-editor/ui/1`, see [this slack thread](https://cq-dev.slack.com/archives/C03RRV5M9B9/p1695059575115709) for more details.

Reply:
Thanks!